Packs/CommunityCommonDashboards/Scripts/XMetrics/XMetrics.py

```diff
@@ -83,14 +83,14 @@ def BuildEffortReductionTable(incidents, efforts) -> dict:
         for month, val in v.items():
             ival = int(val)
             if inctype in efforts:
-                autoeffort = int(ival * efforts[inctype][1])
+                deleff = efforts[inctype][0] - efforts[inctype][1]
+                effreduced = int(ival * deleff)
             else:
-                autoeffort = 0
+                effreduced = 0
             if inctype not in table:
                 table[inctype] = [0] * 12
-            if autoeffort != 0:
-                table[inctype][monthindex] = autoeffort
-                total[monthindex] += autoeffort
+            table[inctype][monthindex] = effreduced
```

Comment:
can you please explain why did you removed the if statement:
if autoeffort != 0: